server/channels/app/platform/cluster_handlers.go

```diff
@@ -60,7 +60,9 @@ func (ps *PlatformService) ClusterUpdateStatusHandler(msg *model.ClusterMessage)
 		ps.logger.Warn("Failed to decode status from JSON")
 	}
 
-	ps.statusCache.SetWithDefaultExpiry(status.UserId, status)
+	if err := ps.statusCache.SetWithDefaultExpiry(status.UserId, status); err != nil {
+		ps.logger.Warn("Failed to store the given key and value")
```

Comment:
Let's also log the error here and in the rest of the file using `mlog.Err`.
```suggestion
		ps.logger.Warn("Failed to store the given key and value", mlog.Err(err))
```

Reply:
I've resolved the issue.